lbz/exceptions.py

```diff
@@ -33,87 +34,291 @@ def get_response(self, request_id: str) -> Response:
         return Response(
             {"message": self.message, "request_id": request_id},
             status_code=self.status_code,
-            headers={"Content-Type": "application/json"},
```

Comment:
Why did you remove that? Or what am I skipping around this change? 🤔

Reply:
its added by default - unnecesery